airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -819,11 +819,11 @@ def process_executor_events(
                 ti.pid,
             )
 
-            if (active_ti_span := cls.active_spans.get(ti.key)) is not None:
+            if (active_ti_span := cls.active_spans.get(ti.try_id)) is not None:
```

Comment:
Sorry, this should now be `ti.id` -- it _was_ ti.try_id, but a few days ago I merged a change where this column was deleted and `ti.id` is the 'unique-per-attempt' ID value we should use now.

Reply:
No worries, I'll update the PR accordingly.